apps/src/templates/SchoolInfoConfirmationDialog.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {Component, PropTypes} from 'react';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
+import Button from './Button';
+import SchoolInfoInterstitial from '../lib/ui/SchoolInfoInterstitial';
+import i18n from '@cdo/locale';
+
+export const styles = {
+  button: {
+    marginTop: 30,
+    marginLeft: 290
+  }
+};
+
+class SchoolInfoConfirmationDialog extends Component {
+  static propTypes = {
+    schoolName: PropTypes.string,
+    onUpdate: PropTypes.func,
+    onConfirm: PropTypes.func,
+    isOpen: React.PropTypes.bool.isRequired
```

Comment:
Other than isOpen, should any of the other props be required? I could see if being reasonable that we wouldn't allow a state where onUpdate/onConfirm are not valid.

Reply:
Yes, there are other props that are required.  Validation will be updated in the follow-up PR.